connect/runtime/src/test/java/org/apache/kafka/connect/integration/OffsetsApiIntegrationTest.java

```diff
@@ -801,6 +806,11 @@ public void testResetSinkConnectorOffsetsZombieSinkTasks() throws Exception {
         connect.assertions().assertConnectorAndAtLeastNumTasksAreRunning(connectorName, 1,
                 "Connector tasks did not start in time.");
 
+        // Make sure the tasks' consumers have had a chance to actually form a group
+        // (otherwise, the reset request will succeed because there won't be any active consumers)
+        verifyExpectedSinkConnectorOffsets(connectorName, topic, 1, NUM_RECORDS_PER_PARTITION,
```

Comment:
Looks like this test was already verifying this?

Reply:
And since this commit no longer changes the behavior of the Reset test, does that mean that KAFKA-15224 is not addressed in this commit? Is that flaky failure still possible?

Reply:
Great point, I've removed mention of KAFKA-15224 from this PR.